### PR TITLE
Olga js changes

### PR DESCRIPTION
**Pull request by okavvada**
* star and read me css
* instagram logo
* expand with click on text
* title font size

### CODE REVIEW

File: html/src/Post.js [hunk 1]
```diff
@@ -116,7 +118,13 @@ class Post extends React.Component {
         />
       {stars}
       <div className="text-container read-more-container">
-        <div className="text read-section">
+        <div className={textClasses}
+            onClick={
+              () => {
+                sendMetric('post_expanded', {post_id: this.props.post.id});
```
Comment (jackdreilly) [inline html/src/Post.js]: Can you put this in a single helper function: this.expand, or something, and then change the two locations this is called to use the same function.

You can make it an instance method of the class.

File: html/src/index.css [hunk 2]
```diff
@@ -153,17 +153,27 @@ footer {
...
+  padding-left: 9px;
+  padding-right: 9px;
+  box-shadow: #bdbdbc 2px 2px 8px 0px;
 }
 
 .more-text:hover {
   color:gray;
 }
 
+.read-section.not-expanded:hover p{
```
Comment (jackdreilly) [inline html/src/index.css]:
```suggestion
.read-section.not-expanded:hover p {
```

File: html/src/index.css [hunk 3]
```diff
@@ -333,18 +360,30 @@ header {
     max-width: 120px;
   }
 
+  #instagram-img img{
```
Comment (jackdreilly) [inline html/src/index.css:368]:
```suggestion
  #instagram-img img {
```

File: html/src/index.css [hunk 4]
```diff
@@ -333,18 +360,30 @@ header {
...
+  header h1 {
+    font-size: 3.5vw;
+}
+
   #dropdown .link-block {
     font-size: 7vw;
     padding-bottom: 3vw;
   }
 
+  #instagram-img img{
```
Comment (jackdreilly) [inline html/src/index.css]:
```suggestion
  #instagram-img img {
```

File: html/src/insta.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function InstagramIMG(props) {
+	return (
+		<a
+		  id="instagram-img"		
+		  target="_blank" 
+		  href="http://instagram.com/taxeedee_travels"
+		  rel="noopener noreferrer"
+		  ><img alt="instragram logo" src="/images/ig-logo.png" /></a>
```
Comment (jackdreilly) [inline html/src/insta.js]: fix indent

**Review by jackdreilly — APPROVED** [review]
LGTM Great stuff Olgaki Mou, You're really looking out for the users! 🥇

File: html/src/insta.js (hunk 5 above)
Comment (okavvada) [reply html/src/insta.js]: ok

**Pull request merged**